src/command-registry.coffee

```diff
@@ -18,6 +18,11 @@ SequenceCount = 0
 # command event listeners globally on `atom.commands` and constrain them to
 # specific kinds of elements with CSS selectors.
 #
+# Command names must follow the `namespace:event` pattern, where both parts can
+# be further split into words by hyphens. Typically `namespace` will be the
```

Comment:
:+1: We should probably also enforce that in code somewhere too, but that's a good idea.

(It would be pretty jarring if you installed `my-awesome-package` and all the commands were namespaced as `settings-view` :stuck_out_tongue_closed_eyes:)